### PR TITLE
Fix CI for deprecated set-output

### PR DESCRIPTION
**Pull request by farhangnaderi**
### Solved Problem
The 'set-output' function is deprecated and needs to be replaced. As @mrpollo mentioned I tried to make the changes. 

### Solution
- Replace the deprecated 'set-output' function with the recommended alternative 'echo ::set-output'.

### Changelog Entry
For release notes:

"Replaced deprecated 'set-output' function with 'echo ::set-output' in Ci "

### Alternatives
We could also continue using the deprecated 'set-output' function, but it's recommended to use the new syntax.

### Test coverage
- Unit/integration test: N/A (No changes impacting tests)
- Simulation/hardware testing logs: N/A

### Context
N/A (No specific context or related links for this PR)

### CODE REVIEW

File: .github/workflows/cflite_batch.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: ClusterFuzzLite batch fuzzing
 on:
-  schedule:
-    - cron: '0 6 * * *'  # UTC 6am every day.
+  #schedule:
+   # - cron: '0 6 * * *'  # UTC 6am every day.
```
Comment (mrpollo) [inline .github/workflows/cflite_batch.yml:4]: do you mind cleaning up? plz uncomment

File: .github/workflows/compile_macos.yml [hunk 2]
```diff
@@ -3,43 +3,44 @@ name: MacOS build
...
-    - 'main'
+      - 'main'
   pull_request:
     branches:
-    - '*'
+      - '*'
 
 jobs:
   build:
-    runs-on: macos-10.15
```
Comment (mrpollo) [inline .github/workflows/compile_macos.yml:13]: this is conflicting with a recent PR that came in

**Review by mrpollo — CHANGES_REQUESTED** [review]
@farhangnaderi can you please rollback the whitespace fixes? let's keep this PR exclusive to the set-output change

**Comment by farhangnaderi** [comment]
> @farhangnaderi can you please rollback the whitespace fixes? let's keep this PR exclusive to the set-output change

@mrpollo you may check, pushed a new one, let me know if I am still lost somewhere! :)

**Comment by junwoo091400** [comment]
@farhangnaderi could you remove the whitespace changes?

File: .github/workflows/deploy_all.yml [hunk 3]
```diff
@@ -11,44 +11,36 @@ jobs:
...
         mkdir _metadata || true
         cp parameters.* events/*.xz actuators.json* _metadata
 
-    - uses: jakejarvis/s3-sync-action@master
+    - name: Upload metadata
+      uses: actions/upload-artifact@v2
       with:
-        args: --acl public-read
-      env:
-        AWS_S3_BUCKET: 'px4-travis'
```
Comment (dagar) [inline .github/workflows/deploy_all.yml:48]: Is this change intentional?

Comment (farhangnaderi) [reply .github/workflows/deploy_all.yml:48]: no, me missing it. Corrected I guess!

**Comment by farhangnaderi** [comment]
> @farhangnaderi could you remove the whitespace changes?

did not again? missed some! my bad! must be ok now!

**Pull request closed without merging**